<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenshinImpactNaturalLawClass</class>
 <widget class="QMainWindow" name="GenshinImpactNaturalLawClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天理</string>
  </property>
  <property name="windowIcon">
   <iconset theme="TrayIcon" resource="GenshinImpactNaturalLaw.qrc">
    <normaloff>:/icon/resource/icon/ICON.png</normaloff>:/icon/resource/icon/ICON.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="MainBackgroundImageRect">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>730</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(225, 225, 225)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="GenshinImpactNaturalLaw.qrc">:/BackgroundImage/resource/bg.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="TitleLabelRect">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(20, 20, 20);
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>  天理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StartGame">
    <property name="geometry">
     <rect>
      <x>916</x>
      <y>626</y>
      <width>238</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(255, 199, 9);
color:rgb(134, 77, 29);
border:none;
border-radius:3px;
}
QPushButton:hover {
background-color:rgb(255, 204, 33);
}
QPushButton:pressed {
background-color:rgb(255, 204, 33);
}</string>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TitleMin">
    <property name="geometry">
     <rect>
      <x>1211</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(0, 0, 0);
border:none;
border-image:url(:/TitleButtonImage/resource/titlebutton/bmin0.png)
}
QPushButton:hover {
border-image:url(:/TitleButtonImage/resource/titlebutton/bmin1.png)
}
QPushButton:pressed {
border-image:url(:/TitleButtonImage/resource/titlebutton/bmin1.png)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TitleSet">
    <property name="geometry">
     <rect>
      <x>1172</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(255, 0, 0);
border:none;
border-image:url(:/TitleButtonImage/resource/titlebutton/bset0.png)
}
QPushButton:hover {
border-image:url(:/TitleButtonImage/resource/titlebutton/bset1.png)
}
QPushButton:pressed {
border-image:url(:/TitleButtonImage/resource/titlebutton/bset1.png)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TitleExit">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(0, 0, 0);
border:none;
border-image:url(:/TitleButtonImage/resource/titlebutton/bexit0.png)
}
QPushButton:hover {
border-image:url(:/TitleButtonImage/resource/titlebutton/bexit1.png)
}
QPushButton:pressed {
border-image:url(:/TitleButtonImage/resource/titlebutton/bexit1.png)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1204</x>
      <y>30</y>
      <width>76</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0,38);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Linke_1">
    <property name="geometry">
     <rect>
      <x>1222</x>
      <y>66</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(0,0,0,120);
color:rgb(255, 255, 255);
border-image:url(:/LinkeIcon/resource/LinkeIcon/Bilibili.png);
border:rgb(82, 92, 115);
border-radius:20px;
}
QPushButton:hover {
border:2px solid rgb(255, 210, 132);
}
QPushButton:pressed {
border:2px solid rgb(255, 210, 132);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Linke_2">
    <property name="geometry">
     <rect>
      <x>1222</x>
      <y>136</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(0,0,0,120);
color:rgb(255, 255, 255);
border:rgb(82, 92, 115);
border-radius:20px;
border-image:url(:/LinkeIcon/resource/LinkeIcon/QQ_icon.png);
}
QPushButton:hover {
border:2px solid rgb(255, 210, 132);
}
QPushButton:pressed {
border:2px solid rgb(255, 210, 132);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Linke_3">
    <property name="geometry">
     <rect>
      <x>1222</x>
      <y>206</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(0,0,0,120);
color:rgb(255, 255, 255);
border:rgb(82, 92, 115);
border-radius:20px;
border-image:url(:/LinkeIcon/resource/LinkeIcon/GitHub_icon.png);
}
QPushButton:hover {
border:2px solid rgb(255, 210, 132);
}
QPushButton:pressed {
border:2px solid rgb(255, 210, 132);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Linke_4">
    <property name="geometry">
     <rect>
      <x>1222</x>
      <y>276</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(0,0,0,120);
color:rgb(255, 255, 255);
border:rgb(82, 92, 115);
border-radius:20px;
}
QPushButton:hover {
border:2px solid rgb(255, 210, 132);
}
QPushButton:pressed {
border:2px solid rgb(255, 210, 132);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Linke_5">
    <property name="geometry">
     <rect>
      <x>1222</x>
      <y>346</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(0,0,0,120);
color:rgb(255, 255, 255);
border:rgb(82, 92, 115);
border-radius:20px;
border-image:url(:/LinkeIcon/resource/LinkeIcon/KYJG_yuanshen.png);
}
QPushButton:hover {
border:2px solid rgb(255, 210, 132);
}
QPushButton:pressed {
border:2px solid rgb(255, 210, 132);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>234</y>
      <width>460</width>
      <height>213</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>63</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(200, 200, 200);
border:1px solid rgb(248, 250, 251);
border-radius:0px;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>广告位招租</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>463</y>
      <width>460</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget{
color:rgb(255, 255, 255);
	background:rgba(0, 0, 0, 50);
}
QTabWidget::pane /* The tab widget frame */  
{	
    font-family:&quot;微软雅黑&quot;;
	font-size:14px;
	top:20px;
	border:none;



}

QTabBar::tab{
	font-family:&quot;微软雅黑&quot;;
	font-size:14px;
	width:69px;
	height:38px;
	color:rgb(255, 255, 255);
	background:rgba(0, 0, 0, 100);
border:none;
}
 
QTabBar::tab:selected, QTabBar::tab:hover
{
	color:rgb(255, 210, 132);
	background:rgba(0, 0, 0, 100);
	border-bottom:0px solid rgb(255, 210, 132);
}
 
QTabWidget::tab-bar {  		/*整个最上面的tab栏*/

left:10px;
} 
 QTabWidget.QWidget{
color:rgb(255, 255, 255);
	background:rgba(0, 0, 0, 50);
}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="styleSheet">
      <string notr="true">QWidget{
color:rgb(255, 255, 255);
	background:rgba(0, 0, 0, 50);
}</string>
     </property>
     <attribute name="title">
      <string>活动</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>更新</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GenshinImpactNaturalLaw.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_TitleExit</sender>
   <signal>clicked()</signal>
   <receiver>GenshinImpactNaturalLawClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1268</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>1060</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_TitleMin</sender>
   <signal>clicked()</signal>
   <receiver>GenshinImpactNaturalLawClass</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1220</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>1344</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
